src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
So you are arguing that SF antichess is bad (despite converse evidence), because gaining Elo is tricky? And why would that be a reason to be more critical of patches?

This is turning into an endless discussion. When I argue against one point, you are bringing up a new one, and you have already mentioned like a dozen reasons why you were critical of the PR. To me it feels like you are dodging my questions.

The only point you made that is really related to this PR is that you are afraid that it makes the search less stable. This is a valid concern, since LMR changes can massively change the search tree, and due to that it might well be that this patch would fail at VLTC or VSTC, but that you never asked for even when I proposed it. If (and I highly doubt that, because it would not explain the dodging) that really is the reason why you are so critical of this PR, then why don't you simply stick to this argument and support/validate/falsify it with tests or search results? (Yes, I remember your point about testing for this being too complex [because you are overcomplicating matters in my opinion], but a hypothesis that can not be tested is of no use.)